buildSrc/build.gradle.kts

```diff
@@ -9,8 +9,8 @@ repositories {
 }
 
 dependencies {
-    implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.21")
-    implementation("com.android.tools.build:gradle:7.3.1")
-    implementation("org.jetbrains.dokka:dokka-gradle-plugin:1.7.20")
+    implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.9.20")
```

Comment:
You can reuse version catalog here if keeping versions aligned is the goal.
It could be like:

in `settings.gradle.kts` of buildSrc
```
dependencyResolutionManagement {

    versionCatalogs {
        register("libs") {
            from(files("../gradle/libs.versions.toml"))
        }
    }
}
```
```